docs/dev/cli/saucectl/artifacts/download.md

```diff
@@ -10,9 +10,8 @@ Downloads artifacts for a job from Sauce Labs.
 
 ## Usage
 
-<span className="cli">$ saucectl artifacts download jobID globPattern [flags]</span>
+`$ saucectl artifacts download jobID globPattern [flags]`
```

Comment:
@aliciakrebs Are there any plans to also update the formatting for the command options under Option Details? 

We could remove the span tag and replace it with the inline code format -> replace <span className="cli">--target-dir</span> with `--target-dir`?

Reply:
That would be good. Will it still show up in the right nav? In the yaml configuration guides we have something like: 

## `--target-dir`

Reply:
Yes, it'll be similar to the .yaml.md docs -- it will display like this: 
<img width="254" alt="Screenshot 2023-02-06 at 3 50 00 PM" src="https://user-images.githubusercontent.com/15352964/217082740-e06b275e-25a6-4ab6-bc22-75a56119d944.png">

Reply:
I do the work to update the commands in Option Details in a separate PR.